File: Medium/2026-06-25-3737-Count-Subarrays-With-Majority-Element-I/solution.java
class Solution {
    public int countSubarrays(int[] nums, int target) {
        int count = 0;
        int n = nums.length;
        
        for (int i = 0; i < n; i++) {
            if (nums[i] == target) {
                int majorityCount = 0;
                for (int j = i; j < n; j++) {
                    if (nums[j] == target) {
                        majorityCount++;
                    }
                    if (j - i + 1 > 2 * majorityCount) {
                        count++;
                    }
                }
            }
        }
        
        return count;
    }
}

File: Medium/2026-06-25-3737-Count-Subarrays-With-Majority-Element-I/solution.js
var countSubarrays = function(nums, target) {
    let count = 0;
    let n = nums.length;
    
    for (let i = 0; i < n; i++) {
        if (nums[i] === target) {
            let majorityCount = 0;
            for (let j = i; j < n; j++) {
                if (nums[j] === target) majorityCount++;
                if (majorityCount > (j - i + 1) / 2) {
                    count++;
                }
            }
        }
    }
    
    return count;
};